Kickstarter-iOS/DataSources/SettingsAccountDataSource.swift

```diff
@@ -0,0 +1,25 @@
+import KsApi
+import Library
+
+final class SettingsAccountDataSource: ValueCellDataSource {
+
+  func configureRows() {
+    _ = SettingsAccountSectionType.allCases.map { section -> Void in
```

Comment:
This was a suggestion on one of my PRs from @justinswart , but you might try using `.forEach` instead since we're not actually mutating the values of the array, just performing an action for each of it's items. And then we don't need the discardable variable `_`.